app/views/finance/npq/statements/show.html.erb

```diff
@@ -109,11 +109,13 @@
       </ul>
     </div>
     <div class="finance-panel__summary__downloads">
-      <div class="second govuk-list govuk-!-margin-bottom-0">
-        <p>
-          <%= govuk_link_to "Download declarations (CSV)", finance_npq_statement_assurance_report_path(@statement, format: :csv) %>
+      <div class="duvet">
+        <p class="govuk-body-s">
+          <a class="govuk-link" href="/finance/npq/statements/6eb2363d-a691-4a3e-9717-f7208cfe1536/assurance-report.csv">
```

Comment:
I think we can still use `<%= govuk_link_to "Download declarations (CSV)", finance_npq_statement_assurance_report_path(@statement, format: :csv) %>` here?

Reply:
🤦‍♂️